docs/pages/theming/simple.mdx

```diff
@@ -209,5 +209,5 @@ const theme = {
 This API makes super easy to add or remove button sizes or variants in the
 future.
 
-> All Chakra's component follows this API and we've made call component styles
+> All Chakra components use this API and we've made all component styles
 > available to you in the `chakra/` folder of your project.
```

Comment:
@segunadebayo Is this part of the theme generation command you've built? We should probably expand on what this means.